watcher/registryhandler.go

```diff
@@ -280,6 +302,42 @@ func createCronJobObject(k8sAPI *k8sinterface.KubernetesApi, registry armotypes.
 	return cronjob, nil
 }
 
+func createJobObject(k8sAPI *k8sinterface.KubernetesApi, registry armotypes.ContainerImageRegistry) (*batchv1.Job, error) {
+	template, err := k8sAPI.KubernetesClient.CoreV1().ConfigMaps(armotypes.KubescapeNamespace).Get(context.Background(), registryCronjobTemplate, metav1.GetOptions{})
```

Comment:
the namespace can be dynamic

Reply:
@refaelm92 Take it from the config https://github.com/kubescape/operator/blob/main/config/config.go#L111

Reply:
@refaelm92 Actually I see you used `armotypes.KubescapeNamespace` in other places in the code (in the command watcher) and it should be replaced